app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes, through: :posts
+  has_many :comments, through: :posts
+  def last_recent_posts
+    posts.last(3)
+  end
```

Comment:
- [ ] Because a user page should show 3 recent_posts you can order them descendant like:
````ruby
def recent_post
  posts.order('created_at Desc').limit(3)
end
````